lib/pbench/server/database/models/datasets.py

```diff
@@ -857,8 +846,8 @@ class Metadata(Database.Base):
     # {"dashboard.seen": True}
     DASHBOARD = "dashboard.*"
 
-    # USER is arbitrary data saved on behalf of the owning user, as a JSON
-    # document.
+    # USER is arbitrary data saved with the dataset on behalf of an
+    # authenticated user, as a JSON document.
```

Comment:
You should probably update the example path at line 855 (and maybe 180, 181, 1010, 1043, 1044, and 1047).

Reply:
Nothing semantically wrong with the statement at 855; although, as mentioned before, "contact information" probably isn't the most relevant example anymore, and maybe I'll change them to "user.favorite".